jobs/JGDAS_ATMOS_ANALYSIS_DIAG

```diff
@@ -53,7 +55,9 @@ fi
 # Remove the Temporary working directory
 ##########################################
 cd ${DATAROOT}
-[[ ${KEEPDATA} = "NO" ]] && rm -rf ${DATA}
+if [[ ${KEEPDATA} = "NO" ]]; then
+    rm -rf ${DATA}
```

Comment:
And so on for the others. Recommend just doing a find/replace.
```suggestion
cd "${DATAROOT}" || true
if [[ "${KEEPDATA}" == "NO" ]]; then
    rm -rf "${DATA}"
```

Reply:
Done.